src/components.rs

```diff
@@ -1,5 +1,5 @@
 use bevy::prelude::*;
-use super::prelude::*;
+use crate::prelude::*;
```

Comment:
super::prelude changed to crate::prelude it's more universal, and means the same thing throughout the project.